### PR TITLE
Readme Sample version

### PR DESCRIPTION
**Pull request by BombadilToma**

### CODE REVIEW

**Review by vvdanila — CHANGES_REQUESTED** [review]
Please add a markdown marker

**Review by victoriabotis — APPROVED** [review]

**Review by Alex-M91 — APPROVED** [review]

**Review by raicubogdan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# car-app
-A Python Car App
```
Comment (lugigann) [inline README.md:2]: New line

Comment (Prophets995) [inline README.md:2]: Should be a blue car

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+
+
+The famous 1st Group of SDA Academy proudly presents:
+
+                                            THE CAR-APP Project
+
+The projects goal is none less than changing the world. In this revolutionary project The Tribe of The First will help
+You to  
+
```
Comment (vasibarna) [inline README.md:9]: add a new line

**Pull request merged**